bee-signing/bee-signing-derive/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "bee-signing-derive"
+version = "0.1.0-alpha"
+authors = ["IOTA Stiftung"]
+edition = "2018"
+description = "Derive macros for the bee-signing crate"
```

Comment:
this is related to my comment above. I feel like those proc macros might be useful for other scenarios than just private keys. In a wallet there are different things a user might to hide per default (he might want to hide whatever he personally considers to be kept secret)